plugins/inputs/http_listener_v2/http_listener_v2.go

```diff
@@ -136,6 +147,9 @@ func (h *HTTPListenerV2) Start(acc telegraf.Accumulator) error {
 		h.WriteTimeout = config.Duration(time.Second * 10)
 	}
 
+	// Append h.Path to h.Paths
+	h.Paths = append(h.Paths, h.Path)
+
```

Comment:
Please only add `Path` if it is not empty and not yet contained in `h.Paths` as otherwise you might introduce duplicate entries.